Tasks/ANT/ant.ps1

```diff
@@ -108,9 +108,15 @@ if ($jdkPath)
 }
 
 $buildRootPath = Split-Path $antBuildFile -Parent
-$reportDirectoryName = [guid]::NewGuid()
+$reportDirectoryName = "ReportDirectory"
 $reportDirectory = Join-Path $buildRootPath $reportDirectoryName
 
+if(Test-Path $reportDirectory)
```

Comment:
From preliminary investigation, I observed that the task doesn't complete unless the job picked up by vso.worker is complete.